## Pipenv version:  
2023.10.3

## 1st hash:
sha256:0c275a06c5190c5ce00af0acbb61c06374087949f643ef32d355ece12c4db043

In [1]:
import pickle

In [2]:
def load(filename: str):
    with open(filename, 'rb') as f_in:
        return pickle.load(f_in)

In [3]:
dv=load('dv.bin')

In [4]:
dv

DictVectorizer(sparse=False)

In [4]:
model=load('model1.bin')

In [5]:
client = {"job": "retired", "duration": 445, "poutcome": "success"}

In [6]:
X=dv.transform([client])

In [7]:
y_pred=model.predict_proba(X)[0,1]
y_pred

0.9019309332297606

In [8]:
client = {"job": "retired", "duration": 445, "poutcome": "success"}
X=dv.transform([client])
y_pred=model.predict_proba(X)[0,1]
y_pred

0.9019309332297606

## Q3: The probability is 0.902

## Q4: The probability is 0.14

## Q5: Image size is 147MB